wp-content/themes/pub/wporg-learn-2020/css/vendor/_sensei.scss

```diff
@@ -1,31 +1,173 @@
 .sensei {
-	&:not(.post-type-archive-course):not(.tax-course-category) {
-		section {
-			padding: 0;
-		}
-	}
+  &:not(.post-type-archive-course):not(.tax-course-category) {
+    section {
+      padding: 0;
+    }
+  }
+
+    .course {
+    &:not(.post-type-archive-course) {
+      article.course {
+        &:first-child {
+          padding: 4rem 0;
+        }
+      }
+    }
+
+    article.course {
+      header {
+        border-bottom: 1px solid #eee;
+        margin-bottom: 1rem;
+      }
+
+      p {
+        font-size: 16px;
+      }
+
+      li {
+        font-size: 16px;
+      }
```

Comment:
Sorry I missed this the first time, but I think the way we've split the styles here has resulted in a lot of repetition, where these rules previously grouped elements with the same treatment.

I suggest replacing this whole block with the original styles from the customizer but with the `.sensei` parent selector removed:

```
.course:not(.post-type-archive-course) article.course:first-child,
.lesson #main-content {
	padding: 4rem 0;
}

.course article.course header,
.lesson #main-content header {
	border-bottom: 1px solid #eee;
	margin-bottom: 1rem;
}

.lesson .bbp-breadcrumb {
	display: none;
}

.lesson #main-content header {
	padding-bottom: 1rem;
}

.course article.course p,
.course article.course li,
.lesson #main-content table {
	font-size: 16px;
}

.course article.course 
.wp-block-image,
.lesson #main-content .wp-block-image,
.course article.course 
.wp-block-media-text,
.lesson #main-content .wp-block-media-text {
	margin: 2rem 0;
}

.course article.course hr,
.lesson #main-content hr {
	margin: 2rem 0;
}

h2.wp-block-sensei-lms-course-outline-module__title {
	color: #fff !important;
}

.wp-block-sensei-lms-lesson-properties {
	padding: 1rem;
	border: 1px solid #e7e7e7;
	display: inline-block;
	display: flex;
	justify-content: center;
	margin-bottom: 1rem;
}

.lesson #main-content nav.post-entries {
	width: 100%;
}

.lesson #main-content nav.post-entries div,
.quiz #main-content nav.post-entries div {
	display: block;
	width: 45% !important;
	padding: 1rem;
	margin: 1rem 0;
	font-style: italic;
}

.lesson #main-content nav.post-entries div.fl,
.quiz #main-content nav.post-entries div.fl {
	padding-left: 0;
}

.lesson #main-content nav.post-entries div.fr,
.quiz #main-content nav.post-entries div.fr {
	padding-right: 0;
	text-align: right;
}

.lesson #main-content .sensei-breadcrumb, 
.quiz #main-content .sensei-breadcrumb {
	padding: 1rem;
}

ol#sensei-quiz-list {
	padding-left: 2rem;
}

.sensei-quiz-action .button {
	height: auto !important;
}
```